constants/locales.js

```diff
@@ -2,33 +2,41 @@ module.exports = [
   {
     i18nString: 'ftm-english',
     localeCode: 'en',
+    nativeLang: 'English',
   },
   {
     i18nString: 'ftm-french',
     localeCode: 'fr',
+    nativeLang: 'Français',
   },
   {
     i18nString: 'ftm-german',
     localeCode: 'de',
+    nativeLang: 'Deutsch',
   },
   {
     i18nString: 'ftm-italian',
     localeCode: 'it',
+    nativeLang: 'Italiano',
   },
   {
     i18nString: 'ftm-polish',
     localeCode: 'pl',
+    nativeLang: 'Polski',
```

Comment:
Are we using i18nString anymore? If not, we can remove from here + en.json

Reply:
I don't think we are, I kept just in case